public/app/features/alerting/unified/hooks/useSilenceNavData.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+
+import { NavModelItem } from '@grafana/data';
+
+export function useSilenceNavData() {
+  const { isExact, path } = useRouteMatch();
+  const [pageNav, setPageNav] = useState<Pick<NavModelItem, 'id' | 'text' | 'icon'> | undefined>();
+
+  useEffect(() => {
+    if (path === '/alerting/silence/new') {
+      setPageNav({
+        icon: 'bell-slash',
+        id: 'silence-new',
+        text: 'Add silence',
+      });
+    } else if (path === '/alerting/silence/:id/edit') {
+      setPageNav({
+        icon: 'bell-slash',
+        id: 'silence-edit',
+        text: 'Edit silence',
```

Comment:
this is missing a breadcrumbs bit (so that it get's a breadcrumb when topnav is disabled)

Reply:
thanks @torkelo , I added the missing breadcrumbs